### PR TITLE
Added grafana charts for pools and volumes

### PR DESCRIPTION
**Pull request by Ab-hishek**
#### Special notes for your reviewer:
The following grafana dashboards for volumes and pools are added:
1. clustervolumesdashboard
2. cstorvolumereplicasdashboard
3. localpvdashboard
4. storagepoolclaimdashboard
5. storageclassvolumedashboard
6. Storagepooldashboard
7. volumetiledviewdashboard
8. volumeworkloadreportdashboard

### Localpv dashboard:
Screenshot from 2021-02-13 15-41-39

### clustervolume dashboard:
Screenshot from 2021-02-12 15-20-43

### cstorvolumereplica dashboard:
Screenshot from 2021-02-12 21-57-50

### storageclassvolume dashboard:
Screenshot from 2021-02-12 22-41-47

### storagepoolclaim dashboard:
Screenshot from 2021-02-12 15-15-24

### storagepool dashboard:
Screenshot from 2021-02-13 16-33-03

### volumetitleview dashboard:
Screenshot from 2021-02-12 21-51-06

### volumeworkloadreport dashboard:
Screenshot from 2021-02-12 14-08-09

Refer the following doc for details regarding the dashboards: https://docs.google.com/document/d/1hM9jaPX8OUoAx4exklJq0pLxgvvf-Hh4BJLaZ8V2bFs/edit
 
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/openebs]`)

### CODE REVIEW

**Comment by survivant** [comment]
ohhhhhhhhhhhhhhhhhh... I want to try that so much :)

**Comment by kmova** [comment]
@Ab-hishek - could you also add a README.md to the grafana-charts folder and push the content from the attached google doc into it. 

Also, can you try the following and then share a screenshot. 

=> On new K8s cluster
   => Run OpenEBS 
   => Launch Minio with Local PV
   => Install Prometheus and Grafana
   => Load the local PV dashboards

Similarly, repeat the same for cStor and attach a screen shot. 

Thank you so much for doing this!

**Comment by Ab-hishek** [comment]
Sure. Will do so.

File: grafana-charts/cstorvolumereplicasdashboard.json [hunk 1]
```diff
@@ -0,0 +1,1383 @@
...
+				"icon": "doc",
+				"includeVars": true,
+				"keepTime": true,
+				"tags": [
+				],
+				"targetBlank": true,
+				"title": "report",
+				"tooltip": "",
+				"type": "link",
+				"url": "${MAYA_URL}/reporter/?dashboard=644d0435-03e1-4da3-b768-7968f526cf68"
```
Comment (akhilerm) [inline grafana-charts/cstorvolumereplicasdashboard.json:65]: @Ab-hishek What should we replace here. ?

@survivant I guess this is the part that you mentioned?

File: grafana-charts/Storagepooldashboard.json [hunk 2]
```diff
@@ -0,0 +1,799 @@
...
+    {
+      "icon": "doc",
+      "includeVars": true,
+      "keepTime": true,
+      "tags": [],
+      "targetBlank": true,
+      "title": "report",
+      "tooltip": "",
+      "type": "link",
+      "url": "https://director.mayadata.io/reporter/?dashboard=5d86b1fd-b2e7-4bb2-befa-4dae5b6167d6"
```
Comment (akhilerm) [inline grafana-charts/Storagepooldashboard.json:40]: Shouldn't this URLs be replaced?

**Review by akhilerm — COMMENTED** [review]
given some comments.

File: grafana-charts/cstorvolumereplicasdashboard.json (hunk 1 above)
Comment (Ab-hishek) [reply grafana-charts/cstorvolumereplicasdashboard.json:65]: Yes Akhil. Looking into this.

File: grafana-charts/Storagepooldashboard.json (hunk 2 above)
Comment (Ab-hishek) [reply grafana-charts/Storagepooldashboard.json:40]: Yes, it should be changed.

File: grafana-charts/cstorvolumereplicasdashboard.json (hunk 1 above)
Comment (survivant) [reply grafana-charts/cstorvolumereplicasdashboard.json:65]: yes.

**Comment by survivant** [comment]
There is a change to do.  The filename **Storagepooldashboard.json**   should be in lowercase

the reason is I want to import them from Helm into grafana.. but the filename will trigger this error

````
-Storagepooldashboard": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.co
m', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
````

**Comment by survivant** [comment]
I created a repo https://github.com/survivant/openebs-grafana
to help to test the dashboards from a fresh Prometheus-Grafana installation that will import those dashboard.

**Comment by Ab-hishek** [comment]
The dashboards here won't give any charts. I am currently testing them @survivant . There are many changes to be done in these dashboards. Have made the changes locally.. Once verified fully I'll push them.

**Comment by Ab-hishek** [comment]
Made all the required changes and added the README.md file.

**Review by kmova — APPROVED** [review]

**Comment by Ab-hishek** [comment]
@survivant The dashboards are ready now. And the steps to create the dashboards are also mentioned in the README.md file. You can try it out and let me if anything more needs to be done.

**Pull request merged**